src/main/scala/uk/gov/nationalarchives/omega/api/services/ApiService.scala

```diff
@@ -124,13 +128,14 @@ class ApiService(val config: ServiceConfig) extends Stateful {
     consumer: JmsAcknowledgerConsumer[IO],
     producer: JmsProducer[IO],
     localMessageStore: LocalMessageStore,
+    repository: OmegaRepository,
```

Comment:
For consistency could this be renamed to `omegaRepository` as above?

Reply:
Done.